keyboards/ah/haven80_hotswap/readme.md

```diff
@@ -0,0 +1,20 @@
+#Haven80 Hotswap
+
+![haven 80 hotswap](https://i.imgur.com/lXKf8Ci.png)
+
+
+* Keyboard Maintainer: [Freather](https://github.com/CMMS-Freather)
+* Hardware Supported: PCB, Atmega32u4
+
+Make example for this keyboard (after setting up your build environment):
+
+    make ah/haven80_hotswap:default
+
+Flashing example for this keyboard:
+
+    make ah/haven80_hotswap:default:flash
+
+
```

Comment:
```suggestion
## Bootloader

```

Reply:
> 

edited the file to show the corresponding bootloader